draft-ietf-dprive-dnsoquic.md

```diff
@@ -290,7 +293,9 @@ in {{RFC1035}}.
 
 The client MUST select the next available client-initiated bidirectional stream
 for each subsequent query on a QUIC connection, in conformance with the QUIC
-transport specification {{!RFC9000}}.
+transport specification {{!RFC9000}}. Packet losses and other network events might
+cause queries to arrive in a different order. Servers SHOULD process queries
+as they arrive, as not doing so would cause unnecessary delays.
```

Comment:
Is the different order with respect to the streams being opened or the stream IDs?? 

```suggestion
transport specification {{!RFC9000}}. Packet losses and other network events might
cause queries to arrive in a different order than streams are opened. Servers 
SHOULD process queries as they arrive regardless of stream ID, as not doing
so would cause unnecessary delays.
```

Reply:
Yes, order of stream IDs. Client can send stream 0 and then stream 4, server may very well receive the bytes and FIN of stream 4 before stream 0. Processing out of stream order protects against head of line blocking.